include/tvm/relay/attrs/transform.h

```diff
@@ -139,6 +139,21 @@ struct StridedSliceAttrs : public tvm::AttrsNode<StridedSliceAttrs> {
   }
 };
 
+struct SliceAxisAttrs : public tvm::AttrsNode<SliceAxisAttrs> {
```

Comment:
I think we already have a slice op. Let us just use that one to avoid duplication:

see https://docs.tvm.ai/langref/relay_op.html#tvm.relay.strided_slice

Reply:
The problem of directly using strided_slice to generate slice_axis in the frontend converter is we need to know the input shape to generate arguments for strides_slice. This is the major task in compute of slice_axis. After this is done, we just call strided_slice.

Reply:
I guess we only need to know the input dimension instead of the shape?

Reply:
Yes, input dimension.